scripts/Source/sslActorAlias.psc

```diff
@@ -938,8 +938,8 @@ state Animating
 		Log(ActorName + ": Orgasms["+Orgasms+"] FullEnjoyment ["+FullEnjoyment+"] BaseEnjoyment["+BaseEnjoyment+"] Enjoyment["+Enjoyment+"]")
 		if Config.OrgasmEffects
 			; Shake camera for player
-			if IsPlayer && Game.GetCameraState() >= 8
-				Game.ShakeCamera(none, 0.70, 2.0)
+			if IsPlayer && Config.ShakeStrength > 0 && Game.GetCameraState() >= 8
```

Comment:
This is picky, but you're comparing a float, so it should be `Config.ShakeStrength > 0.0`